spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -173,6 +174,48 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 	}
 
 	@Test
+	public void testAutoCommitWithCompositeStrategy() throws Exception {
+		logger.info("Start auto");
+		Map<String, Object> props = KafkaTestUtils.consumerProps("test1", "true", embeddedKafka);
+
+		DefaultKafkaConsumerFactory<Integer, String> cf =
+				new DefaultKafkaConsumerFactory<Integer, String>(props,
+						new org.apache.kafka.common.serialization.IntegerDeserializer(),
+						new org.apache.kafka.common.serialization.StringDeserializer());
```

Comment:
M-m-m. I think we don't need such a fully qualified class names.
Just `import` should be enough.
Will fix on merge.